lib/ecto/schema.ex

```diff
@@ -195,6 +198,7 @@ defmodule Ecto.Schema do
       import Ecto.Schema, only: [schema: 2]
       @primary_key {:id, :integer, read_after_writes: true}
       @timestamps_type Ecto.DateTime
+      @timestamps_use_usec true
```

Comment:
Practically with MySQL 5.5 it doesn't even round up, just discards the microseconds, so if 5.5 users just go by this default, it will work just fine.

With 5.6 going with this default it will also just work and the only practical difference is that timestamps can be up to half a second "too quick".

So these worst case scenarios for defaults are not bad I think.

And you even documented the limitations in MySQL in the adapter. I could add a paragraph there saying something along the lines of: If you use MySQL 5.6 set usec: false on timestamps or make the types datetime(6) in your migrations.